compose.yaml

```diff
@@ -0,0 +1,55 @@
+services:
+  postgresql:
+    build: images/postgresql
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+      POSTGRES_HOST_AUTH_METHOD: password # TODO: See how to support scram-sha-256 in go
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres:/var/lib/postgresql/data # Persist data
+      # TODO: Check if we can move structure.sql to ./images/postgresql/01_structure.sql, so we can import the whole folder
```

Comment:
Can I move and rename the `structure.php` file to this directory?

Reply:
sure the `structure.sql` in the project isn't used by the project itself but intended to setup the structure